src/GitHub.InlineReviews/Tags/ShowInlineCommentGlyph.xaml

```diff
@@ -4,21 +4,38 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
              mc:Ignorable="d"
-             ToolTipService.ShowDuration="600000" ToolTipService.HasDropShadow="True">
+             ToolTipService.ShowDuration="600000">
 
-    <UserControl.Resources>
-        <!-- This will be the color of an added/deleted word in the diff view or the indicator margin color -->
-        <SolidColorBrush x:Key="DiffChangeBackground" Color="DarkSlateGray" />
-    </UserControl.Resources>
+    <UserControl.Style>
+        <Style TargetType="UserControl">
+            <Style.Triggers>
+                <DataTrigger Binding="{Binding DiffChangeType}" Value="Add">
+                    <Setter Property="Background" Value="{DynamicResource DiffChangeBackground.Add}" />
+                </DataTrigger>
 
-    <UserControl.ToolTip>
-        <ToolTip x:Name="CommentToolTip" />
-    </UserControl.ToolTip>
+                <DataTrigger Binding="{Binding DiffChangeType}" Value="Delete">
+                    <Setter Property="Background" Value="{DynamicResource DiffChangeBackground.Delete}" />
+                </DataTrigger>
 
-    <Grid Background="{DynamicResource DiffChangeBackground}">
+                <DataTrigger Binding="{Binding DiffChangeType}" Value="None">
+                    <Setter Property="Background" Value="{DynamicResource DiffChangeBackground.None}" />
+                </DataTrigger>
+            </Style.Triggers>
+        </Style>
+    </UserControl.Style>
+
+    <Grid>
         <Viewbox>
             <Path Stroke="Black"
-          Data="M13 2H1c-0.55 0-1 0.45-1 1v8c0 0.55 0.45 1 1 1h2v3.5l3.5-3.5h6.5c0.55 0 1-0.45 1-1V3c0-0.55-0.45-1-1-1z m0 9H6L4 13V11H1V3h12v8z"/>
+                  Data="M13 2H1c-0.55 0-1 0.45-1 1v8c0 0.55 0.45 1 1 1h2v3.5l3.5-3.5h6.5c0.55 0 1-0.45 1-1V3c0-0.55-0.45-1-1-1z m0 9H6L4 13V11H1V3h12v8z"/>
         </Viewbox>
     </Grid>
+
+    <UserControl.ToolTip>
+        <ToolTip
+            Background="{DynamicResource GitHubVsToolWindowBackground}"
```

Comment:
You can now access the GitHub dynamic/themed resources from the glyph XAML files (this wouldn't have worked before). You can see I'm using `GitHubVsToolWindowBackground` here.